python_bindings/bindings.cpp

```diff
@@ -369,7 +361,7 @@ class Index {
                       "mult"_a=appr_alg->mult_,
                       "ef_construction"_a=appr_alg->ef_construction_,
                       "ef"_a=appr_alg->ef_,
-                      "has_deletions"_a=appr_alg->has_deletions_,
+                      "num_deleted"_a=appr_alg->num_deleted_,
```

Comment:
Maybe leave it as is for the compatibility (and fire line 525 in case there are deletions)?

Reply:
Done